test/agent/cmd/aws-node-config/main.go

```diff
@@ -0,0 +1,92 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"strings"
+)
+
+const (
+	MTU_CHECK         = "mtu"
+	VETH_PREFIX_CHECK = "veth"
+)
+
+func main() {
+	validate := ""
+	prefix := "eni"
+	vethCount := 1
+	mtuVal := 9001
+
+	flag.StringVar(&validate, "validate", "", "name of the test, valid values: mtu, veth")
```

Comment:
As we add more validations to this binary the flag combinations required for a given validation might get confusing.

Reply:
Yes, so only the name of the test is mandatory flag, others are optional and get default value. To avoid confusion
1. We can either throw validation errors for any missing related flags.
OR
2. Have separate binaries for each of these small functionalities.
let me know if you have any ideas to avoid the flag confusion.

Reply:
Validation errors could prevent incorrect input. However, my comment was regarding making it easier to decide what is the right flag combinations. Is it possible to evaluate if getting a JSON input can simplify this?

Reply:
We can pass json object as a serialized string. That way all related flags can be together